### PR TITLE
Fix Bug 1451089 - make context menu styles consistent with doorhangers

### PR DESCRIPTION
**Pull request by rlr**
Before:
screen shot 2018-04-05 at 2 39 32 pm
screen shot 2018-04-05 at 2 39 16 pm

After:
screen shot 2018-04-05 at 2 32 43 pm
screen shot 2018-04-05 at 2 43 01 pm

### CODE REVIEW

File: system-addon/content-src/components/ContextMenu/_ContextMenu.scss [hunk 1]
```diff
@@ -35,19 +35,11 @@
...
 
         &:-moz-any(:focus, :hover) {
-          background: var(--newtab-link-primary-color);
-          color: $white;
-
-          a {
-            color: $grey-90;
-          }
+          background: var(--newtab-contextmenu-highlight-background-color);
+          color: var(--newtab-text-primary-color);
```
Comment (rlr) [inline system-addon/content-src/components/ContextMenu/_ContextMenu.scss]: i guess this style here is probably not needed now.

**Comment by Mardak** [comment]
Amy says use the opaque for both

**Comment by rlr** [comment]
This diff cleaned up nicely. for now. I heard there might be more changes coming to the firefox doorhangers and, therefore, our context menus 😄 

Good to ship?

File: system-addon/content-src/components/ContextMenu/_ContextMenu.scss [hunk 2]
```diff
@@ -35,20 +35,7 @@
...
-          }
-
-          .icon {
-            fill: $white;
-          }
-
-          &:-moz-any(:focus, :hover) {
-            color: $white;
-          }
+          background: var(--newtab-element-active-color);
```
Comment (Mardak) [inline system-addon/content-src/components/ContextMenu/_ContextMenu.scss]: Sorry, update light theme to use the non-transparent `--newtab-element-active-color: $grey-20`

**Comment by as-pine-proxy** [comment]
[Treeherder: e59b57a906fe46b127a929b4695236fb7a71c848](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=e59b57a906fe46b127a929b4695236fb7a71c848)

**Pull request merged**